### PR TITLE
Fix/forge loading

### LINKED ISSUE #208: [BUG] Latest Forge release doesn't load (MC 1.20.1)
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - Minecraft version: 1.20.1
 - Voxelsniper version 8.12.1, 8.12.0
 - Server/client software Forge

**Additional context**

**server log/crash report**
Add your log file here, this helps us see what's wrong
17:02:43.832
game
```
Exception in thread "main" java.lang.module.ResolutionException: Modules voxelsniperforge and org.slf4j export package org.slf4j to module minecraft
at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:815)
at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:736)
at java.base/java.lang.module.Resolver.finish(Resolver.java:380)
at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:75)
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:60)
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.run(Launcher.java:106)
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.Launcher.main(Launcher.java:78)
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
at MC-BOOTSTRAP/cpw.mods.modlauncher@10.0.9/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
at cpw.mods.bootstraplauncher@1.1.2/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:141)
```

### PR DESCRIPTION
**Pull request by Lennart99**
## What type of PR is this? (check all applicable)

- [ ] Feature
- [ ] Refactor
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
- Fixes #208 
- Added Minecraft version to the Forge jar file to avoid confusion in the future: _voxelsniper-8.12.2-forge-1.20.1.jar_

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, a note
on the server software and version(s) this has been tested on, as well as any relevant
images._

## Added/updated tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
  have not been included_
- [ ] I need help with writing tests
- [x] Not needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,4 +1,5 @@
-voxelsniper.version=8.12.1
+voxelsniper.version=8.12.2
+voxelsniper.forge-version=1.20.1
```
Comment (KevinDaGame) [inline gradle.properties:2]: I'd say that `voxelsniper.forge.version` would be more semantic

**Review by KevinDaGame — APPROVED** [review]

File: gradle.properties (hunk 1 above)
Comment (KevinDaGame) [reply gradle.properties:2]: Also, `forge.mc-version` would be even better, since forge-version can also mean the `47.5.4.3` number

Comment (Lennart99) [reply gradle.properties:2]: good point, will change that

Comment (Lennart99) [reply gradle.properties]: https://github.com/KevinDaGame/VoxelSniper-Reimagined/pull/209/commits/8e3c43c8dde648d3b09bcd347e74b9e82764762e

**Pull request merged**